test/chunkToNbt.js

```diff
@@ -4,9 +4,11 @@ const { Vec3 } = require('vec3')
 const assert = require('assert')
 const nbt = require('prismarine-nbt')
 
-const testedVersions = ['1.8', '1.13', '1.14', '1.16', '1.17']
+const todoVersions = ['1.9', '1.10', '1.11', '1.12', '1.15']
```

Comment:
Please revert this, we want to explicitly state which versions we want to test

Reply:
Ok, got it, will do within an hour

Reply:
Should we do something so we don't forget about this when doing support for new versions?

Reply:
Historically we did not forget so I think it's not needed